.github/workflows/quality_check.yml

```diff
@@ -7,7 +7,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: "3.7"
+          python-version: "3.11"
```

Comment:
This change is needed to get the quality checks to work. It should probably have happened before, but I guess no one has needed to change the commit-hooks until now.